Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Make sure gcc et al are present for native linking
+FROM oracle/graalvm-ce:20.2.0-java11
+
+# Intended for: zip -r -j target/scala-2.13/invoicing-api.zip target/scala-2.13/bootstrap
+RUN yum -y install zip
+
+# Install sbt and coursier via www.scala-lang.org/2020/06/29/one-click-install.html
+RUN curl -Lo /opt/cs https://git.io/coursier-cli-linux && chmod +x opt/cs
+RUN echo 'Y' | ./opt/cs setup
```

Comment:
This is a bit gnarly

Reply:
The docs say `cs setup --yes` exists

Reply:
I think the fact that it is asking questions here reminds us that this tool is aimed more at end users than build tools. Would it be simpler to [install just sbt](https://github.com/sbt/sbt/releases/download/v1.4.4/sbt-1.4.4.zip), rather than this whole suite? Given the focused use case I'm not sure whether any of the other tools that the one-click install brings us? No harm either way though.

Reply:
sbt-native-image uses coursier under the hood to download graalvm, so I used it to pre-download graal and bake it into the image. However it was just an optimisation.